FlowCal/plot.py

```diff
@@ -96,16 +95,11 @@ def hist1d(data_list,
         (e.g. FCSData).
     log : bool, optional
         Flag specifying whether the x axis should be in log scale.
-    div : int or float, optional
-        Downscaling factor for the default number of bins. If `bins` is not
-        specified, the default set of bins extracted from
-        ``data_list[i].domain`` contains ``n`` bins, and ``div != 1``,
-        `hist1d` will actually use ``n/div`` bins, covering the same range
-        as ``data_list[i].domain``. `div` is ignored if `bins` is
-        specified.
-    bins : array_like, optional
-        bins argument to pass to plt.hist. If not specified, `hist1d`
-        attempts to extract bins from ``data_list[i].domain``.
+    bins : int or array_like, optional
+        If `bins` is an integer, it specifies the number of bins to use.
+        If `bins` is an array, it specifies the bin edges to use. If `bins`
+        is None or an integer, `hist1d` will attempt to use
+        ``data.hist_bins`` to generate the bins automatically.
```

Comment:
i don't understand how these two are both true:

> If `bins` is an integer, it specifies the number of bins to use.

> If `bins` is None or **an integer**, `hist1d` will attempt to use ``data.hist_bins`` to generate the bins automatically.

you have two different sets of instructions for the same condition. may still be true, but it makes description unclear

how is `data.hist_bins` used if `bins` is integer? it uses `hist_bins` as context? defining the range over which to populate the specified number of bins? again, unclear from description.